FAQ.md

```diff
@@ -0,0 +1,268 @@
+# Frequently Asked Questions (FAQ)
+
+Welcome to the **CDEvents FAQ**! This page provides answers to common questions about CDEvents. Whether you're new to CDEvents or looking to deepen your understanding, you'll find valuable information here. If you don't find the answer you're looking for, please refer to our [documentation](https://cdevents.dev/docs/), [whitepaper](https://cdevents.dev/docs/wpaper/), or explore relevant discussions on [GitHub](https://github.com/cdevents).
+
+---
+
+## Table of Contents
+
+1. [General Questions](#general-questions)
+2. [Technical Questions](#technical-questions)
+3. [Implementation and Integration](#implementation-and-integration)
+4. [Security and Compliance](#security-and-compliance)
+5. [Community and Support](#community-and-support)
+6. [Contributing](#contributing)
+7. [Miscellaneous](#miscellaneous)
+
+---
+
+## General Questions
+
+### **Q: What is CDEvents?**
+**A:** CDEvents is an open specification for Continuous Delivery events, designed to promote interoperability between tools in the software production ecosystem. It standardizes the format of events across various phases of the software development lifecycle, enabling seamless communication and integration between different CI/CD tools.
+
+---
+
+### **Q: Why was CDEvents created?**
+**A:** CDEvents was created to address the fragmentation in the Continuous Delivery (CD) toolchain. By providing a common event specification, it allows different tools and platforms to communicate more effectively, reducing integration complexity and preventing vendor lock-in. This fosters a more flexible and adaptable CD pipeline.
+
+---
+
+### **Q: Who is behind CDEvents?**
+**A:** CDEvents is developed and maintained by a collaborative working group comprising industry experts, open-source contributors, and representatives from various organizations committed to enhancing Continuous Delivery practices.
+
+---
+
+### **Q: How does CDEvents differ from other event specifications?**
+**A:** Unlike proprietary event formats or tool-specific integrations, CDEvents is an open, vendor-neutral specification focused solely on Continuous Delivery events. It emphasizes interoperability, flexibility, and ease of adoption across diverse toolchains and environments.

```

Comment:
```suggestion
**A:** Unlike proprietary event formats or tool-specific integrations, CDEvents is an open, vendor-neutral specification focused solely on SDLC events. It emphasizes interoperability, flexibility, and ease of adoption across diverse toolchains and environments.

```

Reply:
By saying 'Continuous Delivery' here, it almost implies that are all the events we have. This is solving SDLC interoperability in the CI/CD space, but might not necessarily by CD or CI.

Reply:
Yes sir,
Will change